draft-ietf-ppm-dap.md

```diff
@@ -1440,21 +1470,17 @@ struct {
 } InputShareAad;
 ~~~
 
-The Leader responds to well-formed requests with HTTP status code 201
-Created. Malformed requests are handled as described in {{errors}}.
-Clients SHOULD NOT upload the same measurement value in more than one report if
-the Leader responds with HTTP status code 201 Created.
```

Comment:
This unhelpfully restates the semantics of 201, plus we no longer specifically require that status anyway